assignments/prototypes.js

```diff
@@ -32,7 +48,19 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid(Hstats){
+  GameObject.call(this, Hstats);
```

Comment:
Character stats gets bound to GameObject on line 33, so since you're passing characterstats on 53, gameobject will already be inherited when binding CharacterStats.